source/index.html

```diff
@@ -5,8 +5,6 @@
 is_homepage: true
 regenerate: true
 feedback: false
-description:
```

Comment:
Shouldn't we update the base template too then where this was being put in?

Reply:
This was use to "override" the head section of the html document, but since it's the same message, there is no need to change it elsewhere.

Reply:
do new lines work inside the attribute of a meta element ?

Reply:
New lines?
<https://deploy-preview-18702--home-assistant-docs.netlify.app/> and <https://deploy-preview-18702--home-assistant-docs.netlify.app/more-info/system-health> is how it looks

Reply:
Those:
![Screenshot 2021-08-02 at 14 36 06](https://user-images.githubusercontent.com/1444314/127926918-dffed25d-32b2-42d1-ad44-c15f7232604d.png)

But looking at the output I don't see them, guess YAML eats them?

Reply:
aaah, `>-` strips that